site/content/docs/howto/idps/configure-supervisor-with-activedirectory.md

```diff
@@ -5,9 +5,9 @@ cascade:
   layout: docs
 menu:
   docs:
-    name: Configure Supervisor With Active Directory
+    name: With Active Directory
     weight: 110
```

Comment:
```suggestion
    weight: 150
```

Reply:
This would put the LDAP guides next to each other, and the AD guide last.

Reply:
👍